views/includes/header.ejs

```diff
@@ -5,8 +5,11 @@
       <% if (!locals.isAuth) { %>
       <li><a href="/signup">Signup</a></li>
       <li><a href="/login">Login</a></li>
-      <% } %> <% if (locals.isAuth) { %>
+      <% } else { %>
       <li><a href="/products">Products</a></li>
+      <% if (locals.isAdmin) { %>
```

Comment:
You could do

```
if (!auth) {
   // do login
} else if (admin) {
   // do admin
} else {
   // do products 
}
```

Reply:
This does not allow admin to view the products page so not changed